backend/globaleaks/handlers/admin/user.py

```diff
@@ -62,6 +62,10 @@ def db_create_user(session, tid, user_session, request, language):
     if not request['username']:
         user.username = user.id = uuid4()
 
+    existing_user = session.query(models.User).filter(models.User.username == user.username).first()
+    if existing_user:
```

Comment:
Please add models.User.tid == tid to ensure to apply this check per tenant, or the check will prevent different tenants' users to use the same username.